client/lib/analytics/utils/cart-to-ga-purchase.ts

```diff
@@ -11,18 +11,19 @@ export type GaPurchase = {
 	items: GaItem[];
 };
 
-export function jetpackCartToGaPurchase(
+export function cartToGaPurchase(
 	orderId: string,
 	cart: ResponseCart,
-	cartInfo: WpcomJetpackCartInfo
+	cartInfo: WpcomJetpackCartInfo,
+	sendToJetpack = false
 ): GaPurchase {
+	const value = sendToJetpack ? cartInfo.jetpackCostUSD : cartInfo.wpcomCostUSD;
```

Comment:
I'd rather create a standalone `cartInfo` object type here (that would have just `costUSD` property, and `cartItems`), rather than pass an additional `sendToJetpack` flag